config/config.go

```diff
@@ -1514,6 +1515,7 @@ var (
 // OTLPConfig is the configuration for writing to the OTLP endpoint.
 type OTLPConfig struct {
 	PromoteResourceAttributes         []string                  `yaml:"promote_resource_attributes,omitempty"`
+	IgnoreResourceAttributes          []string                  `yaml:"ignore_resource_attributes,omitempty"`
```

Comment:
As commented on earlier, I am of the opinion that promoting all resource attributes should be an explicit configuration parameter, that is mutually exclusive with `PromoteResourceAttributes`. I don't think it's a good idea to implicitly promote all resource attributes if `ignore_resource_attributes` is configured as an empty list versus `nil`.


```suggestion
	PromoteAllResourceAttributes          []string                  `yaml:"promote_all_resource_attributes,omitempty"`
	IgnoreResourceAttributes          []string                  `yaml:"ignore_resource_attributes,omitempty"`
```

Reply:
Agree, I will apply the feedback. Thanks :)